api/distributedcontext/context.go

```diff
@@ -16,34 +16,55 @@ package distributedcontext
 
 import (
 	"context"
-	"runtime/pprof"
-
-	"go.opentelemetry.io/otel/api/core"
 )
 
-type ctxEntriesType struct{}
+type (
```

Comment:
My usual suggestion for this type is to have a single key type and create constants of that type : https://play.golang.org/p/EV9K3AAcv7M

Reply:
Done.